configureWorkspace/configure_python.ts

```diff
@@ -32,6 +32,7 @@ WORKDIR /app
 ADD . /app
 
 # Using pip:
+RUN if [ ! -f requirements.txt ]; then pip3 freeze > requirements.txt; fi
```

Comment:
Should this be added to the commented out sections for pipenv and miniconda as well?